### ISSUE TITLE
Issue with "getter": inline code examples are unusually hard to read

### ISSUE THREAD

**Issue by tomprogers** [mdn/content#3955]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get

#### What information was incorrect, unhelpful, or incomplete?

The inline code examples in this list item are unusually hard to parse:

![Screen Shot 2021-04-08 at 11 27 36 PM](https://user-images.githubusercontent.com/3911927/114128538-31b5af80-98c2-11eb-8ed0-4a4a65c50622.png)

I think it's because both examples are composed mostly of parens/braces, presented _inside a parenthetical_, and (on my Macbook) wrap. It's just a perfect storm.

#### Specific section or headline?

Description ([link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get#description))

#### What did you expect to see?

I think it'd be clearer to present them in a block-formatted chunk. That could be problematic inside a list, though.

#### Did you test this? If so, how?

No.

I'm willing to open a PR but I don't know how to preview the final render, and I'm not familiar enough with MDN's formatting styleguide (which I did check) to know what formatting options are available. I could not make this edit blind, and would need to guess-and-test before committing.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/javascript/reference/functions/get`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/functions/get/index.html
* Last commit: https://github.com/mdn/content/commit/6fe447217e08a28cf17b8896545f98d098582ac3
* Document last modified: 2021-02-19T20:01:33.000Z

</details>

**Comment by sideshowbarker** [comment]
> I'm willing to open a PR but I don't know how to preview the final render

We have a mechanism that generates a preview of the rendered version based on the PR content.

> and I'm not familiar enough with MDN's formatting styleguide (which I did check) to know what formatting options are available. I could not make this edit blind, and would need to guess-and-test before committing

It’s fine to commit and iterate — to do the initial commit to the PR branch, look at the generated preview of the rendered version, and then iterate by making any further changes needed and the pushing to the PR branch.

And if you want, while you’re in the process of tweaking it, you can mark the PR as a draft PR (to indicate it’s not ready for review yet).

But if you’d prefer, I’d also be happy to write up a PR for it myself. It’s up to you.
